client/components/sidebar-navigation/index.jsx

```diff
@@ -2,17 +2,23 @@
  * External dependencies
  */
 import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
 
 /**
  * Internal Dependencies
  */
-import layoutFocus from 'lib/layout-focus';
 import Gridicon from 'components/gridicon';
+import { setLayoutFocus } from 'state/ui/layout-focus/actions';
+
+class SidebarNavigationSection extends React.Component {
+	constructor( props ) {
+		super( props );
+		this.toggleSidebar = this.toggleSidebar.bind( this );
```

Comment:
Oh, I guess the name might not be required. There are several components named `SidebarNavigation` and when testing I couldn't tell which one was this one, so I renamed it to be more specific.

I don't think it was technically broken because `toggleSidebar` didn't need to access `this` until this change, so the binding was not necessary.